hphp/runtime/vm/jit/vasm-instr.h

```diff
@@ -1101,6 +1105,10 @@ struct shlq { Vreg64 s, d; VregSF sf; Vflags fl; }; // uses rcx
 /*
  * arm intrinsics.
  */
+struct csoneb { ConditionCode cc; VregSF sf; Vreg8 t, d; };
```

Comment:
Why not a real csinc{} and use `PhysReg(vixl::zr)` as the second source register?
`Conditional Select Increment returns, in the destination register, the value of the first source register if the condition is TRUE, and otherwise returns the value of the second source register incremented by 1.`